docs/kube-api-conditions.md

```diff
@@ -26,6 +26,14 @@ Assisted Installer handles additional types: `SpecSynced`, `ReadyForInstallation
 |Installed|False|InstallationFailed|The installation has failed: "status_info"|If the cluster status is "error"|
 |Installed|False|InstallationNotStarted|The installation has not yet started|If the cluster is before installation ("insufficient"/"pending-for-input"/"ready")|
 |Installed|False|InstallationInProgress|The installation is in progress: "status_info"|If the cluster is installing ("preparing-for-installation", "installing", "finalizing", "installing-pending-user-action")|
+||||||
+|Failed|True|InstallationFailed|The installation failed: "status_info"|if the cluster status is "error"|
+|Failed|False|InstallationNotFailed|The installation has not failed|If the cluster status is not "error"|
+||||||
+|Stopped|True|InstallationFailed|The installation has stopped due to error|if the cluster status is "error"|
+|Stopped|True|InstallationCancelled|The installation has stopped because it was cancelled|if the cluster status is "cancelled"|
+|Stopped|True|InstallationCompleted|The installation has stopped because it completed successfully|if the cluster status is "installed"|
+|Stopped|False|InstallationNotStopped|The installation is in progress|If the cluster status is not "error", "cancelled" or "installed|
```

Comment:
Regarding the message, do we separate between installation in progress (assisted service started the installation) and the discovery phase? or do we see all the process as installing? 
@avishayt @mhrivnak ?

Reply:
I considered installing only when doing the actual installation.

Reply:
So we need another reason?
|Stopped|False|InstallationNotStarted|The installation is in progress|If the cluster status is (before installation statuses)|

Reply:
Hmm actually I think we can include it in InstallationNotStopped

Reply:
OK, so what should be changed? The message for InstallationNotStopped ?

Reply:
We can change it to "The installation is waiting to start or in progress"

Reply:
Done